### DISCUSSION TITLE
Block submission failed with: bad-cb-payee in Yerbas and Bitoreum

### DISCUSSION THREAD

**Discussion by zlunar** [coinfoundry/miningcore#1505] [General]
Hi,

Yerbas (YERB) and Bitoreum (BTRM) working and it does not extract any block due to the error in masternode payment
CSmartnodePayments::IsTransactionValid -- ERROR failed to find expected payee.

Best Regards.

**Comment by oliverw** [comment]
That's the reason I'm going to phase out all coins but the major ones into community maintained packages.

**Comment by minershaven** [comment]
Copy and paste those coins info out of your coins.json file.  Willing to bet the founders reward info is missing

**Reply by minershaven** [reply]
BTRM should look like this. 

"bitoreum": {
        "name": "Bitoreum",
        "canonicalName": "Bitoreum",
        "symbol": "BTRM",
        "family": "bitcoin",
        "website": "https://bitoreum.org/",
        "market": "https://www.coingecko.com/en/coins/bitoreum",
        "twitter": "https://twitter.com/bitoreum",
        "discord": "https://discord.gg/E3E9Ef3kZT",
        "coinbaseHasher": {
            "hash": "sha256d"
        },
        "headerHasher": {
            "hash": "ghostrider"
        },
        "blockHasher": {
            "hash": "reverse",
            "args": [
                {
                    "hash": "sha256d"
                }
            ]
        },
        "hasFounderFee": true,
        "hasMasterNodes": true,
        "foundersRewardAddress": [
            "BanxgMPcMpXnuWQ2ogfQqEkwwVtjhAhXBR"
        ],
        "shareMultiplier": 65536,
        "explorerBlockLink": "https://explorer.bitoreum.org/block/$hash$",
        "explorerTxLink": "https://explorer.bitoreum.org/tx/{0}",
        "explorerAccountLink": "https://explorer.bitoreum.org/address/{0}"
